status/flowcells.py

```diff
@@ -26,7 +26,14 @@ def list_flowcells(self):
         xfc_view = self.application.x_flowcells_db.view("info/summary",
                                                      descending=True)
         for row in xfc_view:
-            temp_flowcells[row.key] = row.value
+            if len(row.value['startdate']) < 7:
+                row.value['startdate'] = "20{}-{}-{}".format(row.value['startdate'][0:2],row.value['startdate'][2:4],row.value['startdate'][4:6]) 
+                temp_flowcells[row.key] = row.value
```

Comment:
I think you can just assign to this value directly?

Reply:
temp_flowcells[row.key] also needs to be assigned not just row.value['startdate'] being changed. See the code that was there before:
```
temp_flowcells[row.key] = row.value
```

Reply:
How about using `datetime` as it is already imported ? something like would do

```
row.value['startdate'] = datetime.strptime(row.value['startdate'], "%y%m%d").strftime("%Y-%m-%d")
```

Reply:
This is the proper (non-lazy) way to do it 😉